src/anomalib/data/video/ucsd_ped.py

```diff
@@ -111,7 +108,7 @@ def get_mask(self, idx: int) -> np.ndarray | None:
         mask_frames = sorted(Path(mask_folder).glob("*.bmp"))
         mask_paths = [mask_frames[idx] for idx in frames.int()]
 
-        return np.stack([cv2.imread(str(mask_path), flags=0) / 255.0 for mask_path in mask_paths])
+        return torch.stack([read_image(mask_path, as_tensor=True) for mask_path in mask_paths]).squeeze()
```

Comment:
would it be an idea to have `read_mask` function that is similar to `read_image` but also does the `squeeze()` stuff. I see that `squeeze` is also done in the base dataset. Doing that in the function would be safer.